apps/editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -343,6 +352,65 @@ interface ComponentProps {
   nodeId: NodeId;
 }
 
+function PropertyInformation(props: ComponentProps) {
+  const { data: blpuCodes } = useQuery(FETCH_BLPU_CODES, {
+    client: publicClient,
+  });
+
+  const propertyTypeVal = props.passport.data?.["property.type"]?.[0];
+
+  return (
+    <>
+      <Box component="dt">{PROPERTY_INFORMATION_DT}</Box>
+      <Box component="dd">
+        {find(blpuCodes?.blpu_codes, { value: propertyTypeVal })?.description || propertyTypeVal || "Unknown"}
+      </Box>
+    </>
+  );
+}
+
+function PlanningConstraints(props: ComponentProps) {
+  // If we weren't able to fetch external data, omit this component from Review
+  //   Any constraints will instead have been manually prompted (and already captured on Review) as questions or checklists
+  const fetchedConstraints = props.passport.data?.["_constraints"];
+  if (!fetchedConstraints) {
+    return undefined;
+  }
+
+  const applicableConstraints = props.passport.data?.["property.constraints.planning"];
+  if (!applicableConstraints?.length) {
+    return (
+      <>
+        <Box component="dt">{PLANNING_CONSTRAINTS_DT}</Box>
+        <Box component="dd">{`No applicable planning constraints found for this property`}</Box>
+      </>
+    );
+  }
+
+  const formattedApplicableConstraints: string[] = [];
+  (applicableConstraints as string[]).forEach((fn) => {
+    fetchedConstraints.forEach((constraintSource: GISResponse) => {
+      // `hasOwnProperty(fn)` will naturally omit/de-duplicate "granular" constraints to their parent
+      if (constraintSource["metadata"].hasOwnProperty(fn)) {
+        formattedApplicableConstraints.push(constraintSource.metadata[fn]["plural"]);
+      }
+    });
+  });
```

Comment:
By iterating over the values of `"property.constraints.planning"`, this will naturally correctly translate any user "overrides" in addition to omitting/de-duping "granular" constraints (eg `Review` page is never going to say the specific constraint entity like "Albert House - Listed Building Grade II" but instead "Listed building outlines" matching the accordion header). 

Eg: 
<img width="738" height="649" alt="Screenshot from 2025-11-08 19-44-16" src="https://github.com/user-attachments/assets/2ead494a-b395-4cdd-a916-76e7d4ab111d" />
Translates to
<img width="797" height="217" alt="Screenshot from 2025-11-08 19-44-30" src="https://github.com/user-attachments/assets/77a33b25-907c-4c7c-8bc0-6e10ce31497b" />